drivers/sensor/wsen_itds/itds_trigger.c

```diff
@@ -96,24 +96,29 @@ int itds_trigger_mode_init(const struct device *dev)
 	struct itds_device_data *ddata = dev->data;
 	const struct itds_device_config *cfg = dev->config;
 
-	ddata->gpio = device_get_binding(cfg->gpio_port);
-	if (!ddata->gpio) {
-		LOG_DBG("Gpio controller %s not found.", cfg->gpio_port);
-		return -EINVAL;
+	/* dts doesn't have GPIO int pin set, so we dont support
+	 * trigger mode for this instance
+	 */
+	if (!cfg->int_gpio.port) {
+		return 0;
+	}
```

Comment:
the API trigger set should also return -ENOTSUP if this holds true (L41)

Reply:
I'm not sure that makes sense in this case.  As the caller `itds_init` is just going to ignore the error code.

Reply:
If we don't do that, application code may try to configure a trigger on an instance that doesn't support it, right?

Reply:
Added check in `itds_trigger_set`